src/utils/prop-types.js

```diff
@@ -66,13 +66,28 @@ export const status = shape({
   runs,
 });
 
-export const task = shape({
-  metadata: shape({
+export const provisionerActions = arrayOf(
```

Comment:
Can you make a `provisionerAction` of just the shape, and then make this `arrayOf(provisionerAction)`?